recipes/autoboost/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "autoboost" %}
+{% set version = "20.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 363bbf6e80b1982b21ad3de4bd4361f8d1b5223c3071d8b84c384b74e8286980
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - matplotlib
```

Does look like this package requires qt. This would reduce the size of the installation drastically and it is highly recommended.

```suggestion
    - matplotlib-base
```

---

@ocefpaf interestingly, now it worked with the base package.